### PR TITLE
Git cheatsheet: Added Blame, Revert and Rebase commands

### PR DESCRIPTION
**Pull request by mayank7**
Git cheatsheet: Added Blame, Revert and Rebase commands

-----
IA Page: https://duck.co/ia/view/git_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@mayank7 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@mayank7 please make sure you add a link to the Git Cheat Sheet IA Page to your PR description. This is required for your PR to be deployed to Beta for testing

File: share/goodie/cheat_sheets/json/git.json [hunk 1]
```diff
@@ -141,6 +147,15 @@
         }, {
             "val": "Downloads bookmark history and incorporates your changes on top of remote changes",
             "key": "git pull --rebase"
+        }, {
+            "val": "Rebase the current HEAD onto branch",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/git.json]: Lets pull these out into a separate section `"Rebasing"`.

**Comment by mayank7** [comment]
Git cheat sheet IA Page: https://duck.co/ia/view/git_cheat_sheet

**Comment by mayank7** [comment]
Added new section Rebasing and removed the redundant command

File: share/goodie/cheat_sheets/json/git.json [hunk 2]
```diff
@@ -138,9 +144,19 @@
         }, {
             "val": "Downloads bookmark history and incorporates changes",
             "key": "git pull"
-        }, {
+        }],
+        "Rebasing": [{
             "val": "Downloads bookmark history and incorporates your changes on top of remote changes",
             "key": "git pull --rebase"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/git.json]: Lets move this one back into the Synchronizing section.

**Comment by GuiltyDolphin** [comment]
@mayank7 By the way, why do you keep closing your PRs and opening new ones? There's plenty of support for editing existing PRs so mistakes in titles/descriptions shouldn't be an issue.

**Comment by mayank7** [comment]
@GuiltyDolphin I put git pull --rebase into Syncronize changes section as suggested by you.

**Comment by mayank7** [comment]
@GuiltyDolphin I put git pull --rebase into Syncronize changes section and it failed in  continuous-integration/travis-ci/pr . 
I am not getting why that happened, can you please review and help me.
These kind of failure is happening for past 1 day, I don't know the reason, everything is appearing correct to me .

**Comment by GuiltyDolphin** [comment]
@mayank7 Thanks for letting me know! (It appears to be a dependency issue - hold tight).

File: share/goodie/cheat_sheets/json/git.json [hunk 3]
```diff
@@ -2,6 +2,7 @@
     "id": "git_cheat_sheet",
     "name": "Git",
     "description": "Git help",
+    "aliases": ["git", "github"],
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/git.json]: Get rid of this.

Comment (mayank7) [inline share/goodie/cheat_sheets/json/git.json]: @GuiltyDolphin I removed it

**Comment by GuiltyDolphin** [comment]
@mayank7 Move into #2620 if still applicable.

Remember to link the IA Page and @mention the maintainer in the PR description.

**Pull request closed without merging**